### PR TITLE
PMM-10064 inventory list pagination

### PR DESCRIPTION
**Pull request by matejkubinec**
https://jira.percona.com/browse/PMM-10064

Replaces tables in inventory list (Services, Agents, Nodes) with component
from core-ui and adds pagination to it, which should help with UI lag
if there's a lot of instances added.

Screenshots:
image

### CODE REVIEW

**Review by fabio-silva — APPROVED** [review]
🚀

**Comment by fabio-silva** [comment]
Please just check the failing test

File: public/app/percona/inventory/Inventory.test.tsx [hunk 1]
```diff
@@ -97,4 +160,21 @@ describe('Inventory tables', () => {
     // length is 3 because header is also tr
     expect(container.querySelectorAll('tr')).toHaveLength(3);
   });
+
+  fit('Nodes table renders first page only', () => {
```
Comment (fabio-silva) [inline public/app/percona/inventory/Inventory.test.tsx:164]:
```suggestion
  it('Nodes table renders first page only', () => {
```

File: public/app/percona/inventory/Inventory.test.tsx [hunk 2]
```diff
@@ -53,6 +53,46 @@ describe('Inventory tables', () => {
...
+        pmm_agent_id: 'pmm-server',
+        service_id: '/service_id/ab477624-4ee9-49cd-8bd4-9bf3b91628b2',
+        username: 'pmm-managed',
+        status: 'RUNNING',
+      }),
+    };
+
+    const { container } = render(
+      <Table
+        data={InventoryDataService.getAgentModel(response as any)}
```
Comment (fabio-silva) [inline public/app/percona/inventory/Inventory.test.tsx:85]: aren't we missing props to make sure only 25 show up, like `showPagination` ?

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: public/app/percona/inventory/Inventory.test.tsx (hunk 2 above)
Comment (matejkubinec) [reply public/app/percona/inventory/Inventory.test.tsx:85]: Yup, this one looks wrong, I'll fix it.

Comment (matejkubinec) [reply public/app/percona/inventory/Inventory.test.tsx:85]: On second look, the test section is weird, I'll update it.

**Review by fabio-silva — APPROVED** [review]

**Review by solovevayaroslavna — APPROVED** [review]

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**